app/pages/Login/Login.js

```diff
@@ -78,10 +81,11 @@ const Login = ({ location }) => {
           <FormField
             label="Email"
             name="email"
+            onChange={e => setEmail(e.target.value)}
```

Comment:
From Emily: 
> This is fine for this usage, but I wanted to call this out for a good conceptual learning point.
> 
> By introducing state that tracks an input value, we're re-rendering the component on each input change (e.g. each keystroke), which we don't need. Since we still need to access the value outside of the form's submit method, we can avoid the use of state by using a ref to keep this as an "uncontrolled" form!
> 
> Something like ref={(emailInput) => { this.emailInput = emailInput }} on the input with the corresponding ref hook should do the trick. IIRC you'd access the valud by this.emailInput.value, but I haven't tested it.